### DISCUSSION TITLE
Add ability to export all metadata about a project to RDF, including information about simulation runs

### DISCUSSION THREAD

**Discussion by bilalshaikh42** [biosimulations/biosimulations#3983] [Ideas]
We have detailed information about algorithms, parameters, settings for simulations etc in our database. 

We should find a way to encode these into the metadata.rdf files for the projects.  

This could also then be loaded into a potential SPARQL endpoint as mentioned in #2857

**Comment by jonrkarr** [comment]
I think it might be easiest to aggregate all of this information into Virtuoso and then let users query and/or export it from there.

**Reply by bilalshaikh42** [reply]
https://github.com/ad-freiburg/qlever might be a better alternative to Virtuoso
